test/org/sagebionetworks/bridge/TestUtils.java

```diff
@@ -128,13 +128,19 @@ public static void assertValidatorMessage(Validator validator, Object object, St
     }
     
     public static void assertResult(Result result, int statusCode, String message) throws Exception {
+        assertResult(result, statusCode);
+        
         JsonNode node = BridgeObjectMapper.get().readTree(Helpers.contentAsString(result));
         String resultMessage = node.get("message").asText();
-        assertEquals(statusCode, result.status());
         assertEquals("application/json", result.contentType());
         assertEquals(message, resultMessage);
     }
 
+    public static void assertResult(Result result, int statusCode) throws Exception {
+        assertEquals(BridgeConstants.JSON_MIME_TYPE, result.header("Content-Type"));
```

Comment:
Minor: The two assertResult() methods both test content type in a different way. Is it worth making them both look the same for consistency?

Reply:
Yes that first one is a duplicate because that method calls the one below it, removing